website/content/en/docs/installation/_index.md

```diff
@@ -88,7 +88,6 @@ chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/l
 #### Prerequisites
 
 - [git][git_tool]
-- [mercurial][mercurial_tool] version 3.9+
 - [bazaar][bazaar_tool] version 2.7.0+
```

Comment:
No need of Bazaar as well

Reply:
Removed `bazaar`, I've never had it installed. 

```
$ which bazaar
/usr/bin/which: no bazaar in (/home/devel/jesusr/bin:/home/devel/jesusr/.local/bin:/home/devel/jesusr/bin:/home/devel/jesusr/.local/bin:/home/devel/jesusr/bin:/home/devel/jesusr/.local/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/home/devel/jesusr/dev/bin:/home/devel/jesusr/dev/bin:/home/devel/jesusr/dev/bin)
```